src/test/java/core/basesyntax/FruitTest.java

```diff
@@ -0,0 +1,46 @@
+package core.basesyntax;
+
+import core.basesyntax.database.Storage;
+import core.basesyntax.model.Fruit;
+import java.util.Optional;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FruitTest {
+    private static Fruit banana;
+    private static Fruit apple;
+    private static Fruit melon;
+
+    public FruitTest() {
+        banana = new Fruit("banana");
+        apple = new Fruit("apple");
+        melon = new Fruit("melon");
+    }
+
+    @Test
+    public void storeFruits_isOk() {
+        Storage.fruitStorage.put(banana, 20);
+        Storage.fruitStorage.put(apple, 10);
+
+        Optional<Integer> optionalBanana = Optional.ofNullable(Storage.fruitStorage.get(banana));
+        int actual = optionalBanana.get();
+        Assert.assertTrue("apple".equals(apple.getName()));
```

Comment:
Assert.assertEquals("apple", apple.getName());

IDEA suggests changing it

Reply:
done